### DISCUSSION TITLE
GPU does not works

### DISCUSSION THREAD

**Discussion by neuromodern** [neuromodern/VideoFake#1] [Q&A]
GPU (Cuda mode) stop works. Can someone check new dependencies?

**Comment by neuromodern** [comment]
![image](https://github.com/neuromodern/VideoFake/assets/111322514/164443b4-43c7-4958-97b0-199f981aac84)
i manualy set cuda in face_encancer
![image](https://github.com/neuromodern/VideoFake/assets/111322514/0a5bbb86-9320-4dbf-a520-dad9ac321bad)

**Reply by neuromodern** [reply]
i got a solution. it is conflict of CUDA and ONNX. onnxruntime-gpu does not support cuda 12 
i did a manual instaling. and it is works. but i do not have a time for rebuild requirements.txt. who can do a work setup?
![image](https://github.com/neuromodern/VideoFake/assets/111322514/fbf24f5f-c998-4877-b81c-9534dd55ef5a)

**Reply by Kalpit8055** [reply]
I can do it but i don't know how to.

**Reply by bran31877-sudo** [reply]
man i have been using gemini to help me get this to run. its "creating video" but i am skeptical lol. if it works i have the code you need
